engine/runtime-integration-tests/src/test/java/org/enso/interpreter/test/meta/MetaIsATest.java

```diff
@@ -28,15 +29,19 @@ public class MetaIsATest {
 
   @BeforeClass
   public static void prepareCtx() throws Exception {
-    ctx = ContextUtils.createDefaultContext();
+    ctx =
+        ContextUtils.defaultContextBuilder()
+            .option(RuntimeOptions.DISABLE_PRIVATE_CHECK, "true")
+            .build();
     final URI uri = new URI("memory://choose.enso");
     final Source src =
         Source.newBuilder(
                 "enso",
                 """
-    from Standard.Base import Meta, Warning
+    import Standard.Base.Meta_Helpers
+    from Standard.Base import Warning
 
-    check x y = Meta.is_a x y
```

Comment:
Calling `Meta.is_a` doesn't really work with `EnsoMultiValue` - another problem to keep in mind for 
- #11935

Reply:
What do you mean by "doesn't work"? Does it return wrong results, or raise an error?

Reply:
It extracts the first value from `EnsoMultiValue` and dispatches to it, I think.